src/server/dstu2/patient/controllers/patient.controller.js

```diff
@@ -71,7 +73,7 @@ module.exports.getPatientById = (profile, logger) => {
 		return service.getPatientById(req, logger, context)
 			.then((patient) => {
 				if (patient) {
-					res.send(patient);
+					res.send(new Patient(patient));
```

Comment:
Same comment as above, we should do `res.status(200).json(new Patient(patient))`